### ISSUE TITLE
[Feature]: Show terminal output when deploying a stack

### ISSUE THREAD

**Issue by RKLBusinessDevelopment** [mbecker20/komodo#77]
When deploying stacks, there is currently no way to see the output of docker. This is inconvenient, especially for stacks that take a while to deploy (e.g. due to downloading large images). It would be great if Komodo would show the docker output when deploying a stack. Check out Dockge to see what this could look like.

Thank you for a great tool!

**Comment by mbecker20** [comment]
Thanks for the feedback. Several users request this and I think it would improve the user experience. But it is not so easy to do, due to the atomic nature of the Core request to Periphery, which uses http request. 

I have some long term thoughts to change the communication model from Core to Periphery to a poll status based approach, which I think will improve some other parts as well, like be able to recover if the http connection is temporarily broken.

This would also promote more real time logs by being closer to a stream based approach, while likely simpler.

**Comment by Samaze123** [comment]
Hey there! I just wanted to check if there’s any progress on adding deployment logs. It’s the one thing that’s keeping me on Dockge, but I really love Komodo and can see so much potential in it.

**Comment by mpkossen** [comment]
@Samaze123 FWIW, once a container has been deployed I can see the stderr output of the compose up command from within Komodo. It's not "live" like it is in Dockge, but it's something.

**Comment by lutek** [comment]
I agree. Since switching to Komodo - primarily for the centralized Git repository for stacks, resources, etc. I've missed the superior log output when deploying stacks, particularly with large ones. I can't monitor errors.

**Comment by Lenart12** [comment]
@mbecker20 do you have any thoughts on revisiting this? It has been two years with changes to both communication model and integrated terminals.

**Comment by jondricek** [comment]
This also affects `RunBuild`: Docker build output only appears after completion. Periphery buffers the command output until exit, so this seems to share the same underlying Core to Periphery streaming limitation. Would be great if a solution to this would cover build logs too.

**Comment by jondricek** [comment]
Since v2, the original atomic HTTP limitation seems largely removed: [Core to Periphery communication is multiplexed over WebSockets](https://github.com/moghtech/komodo/blob/v2.2.0/bin/periphery/src/connection/mod.rs#L64-L133), [terminal output already streams](https://github.com/moghtech/komodo/blob/v2.2.0/bin/core/src/periphery/terminal.rs#L72-L132), and the UI [subscribes to an update WebSocket](https://github.com/moghtech/komodo/blob/v2.2.0/ui/src/lib/socket.tsx#L57-L102).

The remaining work appears to be:

- Read build/deploy stdout and stderr incrementally instead of using output()/wait_with_output().
- Send chunks over the existing channelized transport.
- Have Core redact, optionally persist, and publish partial output.
- Append chunks in the update UI—or trigger throttled refetches.

Of course I'm armchair quarterbacking and am no expert here - just wanted to jot down some notes while I was diving into this since I would love to benefit from it, even though I realize it might be even harder than I'm making it out to be.
